### PR TITLE
docs: Media Chrome Themes docs

### PR DESCRIPTION
**Pull request by luwes**
This adds a first pass of the Themes documentation. It's still a work in progress but any tips or feedback welcome.

https://media-chrome-docs-git-fork-luwes-docs-themes-mux.vercel.app/en/themes

### CODE REVIEW

File: docs/src/pages/en/themes.md [hunk 1]
```diff
@@ -0,0 +1,416 @@
...
+
+Media Chrome provides us with [Web Components](https://developer.mozilla.org/en-US/docs/Web/Web_Components) 
+that are easy to [style via CSS](./styling.md) which is great for media players 
+that are embedded in your own webpage or require less portability. However it's 
+often the case that a media player will be used by 3rd parties or 
+maybe the player needs to support different layouts and styles depending on
+the context.
+
+Themes provide a great solution for changing the look and feel of your player,
+and wrap your media controls up in nice and portable package.
```
Comment (AdamJaggard) [inline docs/src/pages/en/themes.md:15]:
```suggestion
and wrap your media controls up in a nice and portable package.
```

File: docs/src/pages/en/themes.md [hunk 2]
```diff
@@ -0,0 +1,416 @@
...
+Note that for most basic themes it's unlikely you'll even need this.
+
+
+### Variables
+
+The most basic tag type is the variable. A `{{name}}` tag in a basic template 
+will try to find the name key in the current state that is derived from 
+the `<media-theme>` attributes. Variables are escaped by default.
+
+If you like to provide a fallback for a variable that might be empty,
```
Comment (AdamJaggard) [inline docs/src/pages/en/themes.md:123]:
```suggestion
If you would like to provide a fallback for a variable that might be empty,  
```

File: docs/src/pages/en/themes.md [hunk 3]
```diff
@@ -0,0 +1,416 @@
...
+  ></video>
+</media-theme>
+
+[![Edit Media Chrome Vars Theme](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/media-chrome-vars-theme-nejd49?fontsize=14&hidenavigation=1&theme=dark)
+
+In the example above the `videotitle` attribute is provided to the `<media-theme>` 
+element and this is then rendered in the template.
+
+> It's recommended not to use the native `title` attribute because this will add
+add a tooltip on mouse hover.
```
Comment (AdamJaggard) [inline docs/src/pages/en/themes.md:168]: two "add"s
```suggestion
a tooltip on mouse hover.
```

File: docs/src/pages/en/themes.md [hunk 4]
```diff
@@ -0,0 +1,416 @@
...
+`partial` attribute which is set to a unique name in your theme.
+
+```html
+<template partial="PlayButton">
+  <media-play-button></media-play-button>
+</template>
+```
+
+These can then be used in other places in the theme with a partial variable 
+like so `{{>PlayButton}}`. They can also accept params by adding them after
```
Comment (AdamJaggard) [inline docs/src/pages/en/themes.md:198]:
```suggestion
like so `{{>PlayButton}}`. They can also accept parameters by adding them after 
```

File: docs/src/pages/en/themes.md [hunk 5]
```diff
@@ -0,0 +1,416 @@
...
+</template>
+```
+
+These can then be used in other places in the theme with a partial variable 
+like so `{{>PlayButton}}`. They can also accept params by adding them after 
+the var name `{{>PlayButton section="center"}}`.
+
+### Conditionals
+
+When a single element or a multiple elements need to be left out from
```
Comment (AdamJaggard) [inline docs/src/pages/en/themes.md:203]:
```suggestion
When elements need to be left out from 
```

File: docs/src/pages/en/themes.md [hunk 6]
```diff
@@ -0,0 +1,416 @@
...
+These can then be used in other places in the theme with a partial variable 
+like so `{{>PlayButton}}`. They can also accept params by adding them after 
+the var name `{{>PlayButton section="center"}}`.
+
+### Conditionals
+
+When a single element or a multiple elements need to be left out from 
+the theme when certain conditions are met it's possible to use a conditional. 
+A conditional is also defined by an inner template with an `if` attribute. 
+The value of this `if` attribute can be a simple equality check or just a empty check.
```
Comment (AdamJaggard) [inline docs/src/pages/en/themes.md:206]:
```suggestion
The value of this `if` attribute can be a simple equality check or just an empty check.
```
Also, what is mean't by an "empty" check?

**Review by AdamJaggard — COMMENTED** [review]
minor grammar stuff

**Review by dylanjha — APPROVED** [review]

**Pull request merged**